mysql-test/suite/innodb/t/log_encrypt_kill2.test

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE tne_1(c1 INT, c2 varchar(2000)) ENGINE = InnoDB;
+INSERT INTO tne_1 VALUES (1,REPEAT('a',1990)),(2,REPEAT('b',1990)),(100,REPEAT('c',1990));
+SELECT c1,LEFT(c2,10) FROM tne_1;
+--source include/kill_mysqld.inc
+
+--echo # Starting server with keyring plugin
+--let $restart_parameters=restart: --early-plugin-load="keyring_file=$KEYRING_PLUGIN" --loose-keyring_file_data=$MYSQL_TMP_DIR/mysecret_keyring $KEYRING_PLUGIN_OPT --innodb_redo_log_encrypt=ON
+--replace_regex /\.dll/.so/
+--source include/start_mysqld_no_echo.inc
```

Comment:
since this test is about recovery. May be add grep/search_file_in_pattern that recovery is initiated.
Look for some InnoDB message that prints recovery msg.